cmd/crictl/main_unix.go

```diff
@@ -20,5 +20,5 @@ package main
 
 const (
 	defaultConfigPath      = "/etc/crictl.yaml"
-	defaultRuntimeEndpoint = "unix:///var/run/dockershim.sock"
+	defaultRuntimeEndpoint = "unix:///var/run/dockershim.sock unix:///run/containerd/containerd.sock unix:///run/crio/crio.sock"
```

Comment:
Yeah, was thinking the same thing but decided to go with the string instead.

I agree that it would be safer and more robust code to have a separate array with the different defaults.

Reply:
Updated